template/src/features/counter/Counter.js

```diff
@@ -21,14 +21,18 @@ export function Counter() {
           className={styles.button}
           aria-label="Increment value"
           onClick={() => dispatch(increment())}
+          data-testid="counter-increment"
         >
           +
         </button>
-        <span className={styles.value}>{count}</span>
+        <span className={styles.value} data-testid="counter-value">
+          {count}
+        </span>
```

Comment:
In a similar vein, but slightly more complicated, you do this to avoid `data-testid`:
```suggestion
        <span className={styles.value}>
          <span className="sr-only">Count is </span>
          {count}
        </span>
```

and then query by:`screen.getByText(/Count is \d+/i)`

You'd have to add this though: https://gist.github.com/ffoodd/000b59f431e3e64e4ce1a24d5bb36034#file-style-css-L30